system/async/tasks/ScheduledTask.cfc

```diff
@@ -414,19 +414,19 @@ component accessors="true" {
 
 			// After Interceptor
 			if ( isClosure( variables.afterTask ) || isCustomFunction( variables.afterTask ) ) {
-				variables.afterTask( this, variables.stats.lastResult );
+				variables.afterTask( this, (!isNull(variables.stats.lastResult)) ?:"" );
```

Comment:
I don't think you need the is null check since you are using the elvis operator my friend. Also, thinking about it let's juse use the safety navigation operator.

```
variables.afterTask( this, variables.stats?.lastResult );
```

That should safely navigate and return a result or null for us

Reply:
Got it, will change to safe-navigation operator syntax.